pages/common/gimp.md

```diff
@@ -1,25 +1,25 @@
 # gimp
 
-> An image manipulation and paint program.
+> GNU image manipulation program.
 > More information: <https://docs.gimp.org/en/gimp-fire-up.html#gimp-concepts-running-command-line>.
 
 - Launch GIMP:
 
 `gimp`
 
-- Launch but don't show the splash screen:
+- Launch, but don't show the splash screen:
 
 `gimp --no-splash`
 
-- Start a new instance, even if there is already a running one:
+- Always start a new instance:
```

Comment:
In light of my previous comment above, I'd be inclined to actually double down on the existing description:

```suggestion
- Start a new GIMP instance, even if there is already a running one:
```